WooCommerce/WooCommerceScreenshots/Utils/XCTest+Extensions.swift

```diff
@@ -0,0 +1,169 @@
+import XCTest
+
+var isIPhone: Bool {
+    return UIDevice.current.userInterfaceIdiom == .phone
+}
+
+var isIpad: Bool {
+    return UIDevice.current.userInterfaceIdiom == .pad
+}
+
+let navBackButton = XCUIApplication().navigationBars.element(boundBy: 0).buttons.element(boundBy: 0)
+
+extension XCUIElement {
+    /**
+     Removes any current text in the field
+     */
+    func clearTextIfNeeded() -> Void {
+        let app = XCUIApplication()
+
+        self.press(forDuration: 1.2)
+        app.keys["delete"].tap()
+    }
+
+    /**
+     Removes any current text in the field before typing in the new value
+     - Parameter text: the text to enter into the field
+     */
+    func clearAndEnterText(text: String) -> Void {
+        clearTextIfNeeded()
+        self.tap()
+        self.typeText(text)
+    }
+
+    var stringValue: String? {
+        return self.value as? String
+    }
+}
+
+extension XCTestCase {
+
+//    public func setUpTestSuite() {
+//        super.setUp()
+//
+//        // In UI tests it is usually best to stop immediately when a failure occurs.
+//        continueAfterFailure = false
+//
+//        let app = XCUIApplication()
+//        app.activate()
+//
+//        // Media permissions alert handler
+//        systemAlertHandler(alertTitle: "“WordPress” Would Like to Access Your Photos", alertButton: "OK")
+//    }
```

Comment:
Is there a reason to leave this (and other extensions that don't apply to the Woo app) in here? Not a huge deal, though; we can also clean this up when we work on abstracting out the UI test helpers that apply across apps.

Reply:
I wanted to leave this in here because the upcoming products features will allow updating products, which will include managing photos, so it'd be quick to just flip it on :) 

That said, I'm not set on having it, we could just copy it again from WP later on?

Reply:
Good thought. :) Let's leave it, then, since I'll also likely want a `setUpTestSuite()` method to reuse when I start building out more UI tests soon and this will make it easy to enable/adjust the pieces that I want.